lib/mas/cms/entity/article.rb

```diff
@@ -2,7 +2,16 @@ module Mas::Cms
   class Article < Page
     Alternate = Struct.new(:title, :url, :hreflang)
 
-    attr_accessor :type, :slug, :identifier, :title, :description, :body, :categories, :related_content, :supports_amp
+    attr_accessor :type,
+                  :slug,
+                  :identifier,
+                  :title,
+                  :description,
+                  :body,
+                  :categories,
+                  :related_content,
+                  :supports_amp,
+                  :extra_blocks
```

Comment:
can we think of a more descriptive name?

Reply:
I really thought "blocks_without_content" but sounds awful. This one was the best I can think of. :\

Reply:
My concern is that we come back someday to this and wonder what extra_blocks are. `non_ content_blocks`?